examples/debug/carto_vl/style-guide.ipynb

```diff
@@ -0,0 +1,2382 @@
+       "\n",
+       "    function setPopupsClick(tempPopup, interactivity, popupHeader, values) {\n",
+       "      interactivity.off('featureHover', (event) => {\n",
+       "          updatePopup(tempPopup, event, popupHeader, values)\n",
+       "      });\n",
+       "      interactivity.on('featureClick', (event) => {\n",
+       "          updatePopup(tempPopup, event, popupHeader, values)\n",
+       "      });\n",
+       "    }\n",
+       "    \n",
+       "    function setPopupsHover(tempPopup, interactivity, popupHeader, values) {\n",
+       "      interactivity.off('featureClick', (event) => {\n",
+       "          updatePopup(tempPopup, event, popupHeader, values)\n",
+       "      });\n",
+       "      interactivity.on('featureHover', (event) => {\n",
+       "          updatePopup(tempPopup, event, popupHeader, values)\n",
+       "      });\n",
+       "    }\n",
+       "\n",
+       "    function updatePopup(layer_popup, event, popupHeader, values) {\n",
+       "      if (event.features.length > 0) {\n",
+       "        const feature = event.features[0];\n",
+       "        const variables = values && values.length\n",
+       "          ? Object.keys(feature.variables).filter(variable => values.include(variable))\n",
+       "          : Object.keys(feature.variables);\n",
+       "\n",
+       "        let popupHTML = popupHeader ? `<h1>${popupHeader}</h1>` : ``;\n",
+       "        \n",
+       "        Object.keys(feature.variables).forEach((name) => {\n",
+       "            const variable = feature.variables[name];\n",
+       "\n",
+       "            if (variable) {\n",
+       "              console.log('!!! variable', variable);\n",
+       "              popupHTML += `\n",
+       "                <h3 class=&quot;h3&quot;>${name}</h3>\n",
+       "                \n",
+       "              `;\n",
+       "            }\n",
+       "            // <p class=&quot;description open-sans&quot;>${JSON.stringify(variable.value)}</p>\n",
+       "        });\n",
+       "        \n",
+       "        layer_popup\n",
+       "            .setLngLat([event.coordinates.lng, event.coordinates.lat])\n",
+       "            .setHTML(`<div>${popupHTML}</div>`);\n",
+       "        \n",
+       "        if (!layer_popup.isOpen()) {\n",
+       "          layer_popup.addTo(map);\n",
+       "        }\n",
+       "      } else {\n",
+       "        layer_popup.remove();\n",
+       "      }\n",
+       "    }\n",
+       "  });\n",
+       "}\n",
+       "\n",
+       "responsive.addEventListener('ready', onReady);\n",
+       "    </script>\n",
+       "  </body>\n",
+       "  </html>\n",
+       "\">\n",
+       "\n",
+       "</iframe>"
+      ],
+      "text/plain": [
+       "<cartoframes.carto_vl.maps.map.Map at 0x116761080>"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Layer ordering from bottom to top\n",
```

Comment:
What is the reason to change the order? I find this very confusing. This will be totally different than any other vis library from CARTO.

Reply:
this is good feedback. as noted in this issue https://github.com/CartoDB/cartoframes/issues/555 it seemed counterintuitive but that is a good reminder that we'd be breaking convention.

Reply:
Then I would keep the current state